### PR TITLE
PMM-9752 create a FB

### PR DESCRIPTION
**Pull request by ademidoff**
https://jira.percona.com/browse/PMM-9752

- [x] https://github.com/percona/pmm-update/pull/193
- [x] https://github.com/percona/pmm-server/pull/347
- [x] https://github.com/Percona-Lab/jenkins-pipelines/pull/1461

### CODE REVIEW

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-2404-47ee979
client docker - perconalab/pmm-client-fb:PR-2404-47ee979
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2404-47ee979.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-2404-47ee979&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2404-47ee979.tar.gz

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/9503/

**Comment by JNKPercona** [comment]
UI tests have failed, Please check: UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/13150/

**Comment by JNKPercona** [comment]
This build is successful and can be tested.
server docker - perconalab/pmm-server-fb:PR-2404-53b747c
client docker - perconalab/pmm-client-fb:PR-2404-53b747c
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2404-53b747c.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-2404-53b747c&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2404-53b747c.tar.gz

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/9538/

**Comment by JNKPercona** [comment]
UI tests have failed, Please check: UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/13262/

File: ci-default.yml [hunk 1]
```diff
@@ -1,130 +1,136 @@
...
-  path: sources/pmm-qa/src/github.com/percona/pmm-qa
-  url: https://github.com/percona/pmm-qa
-  component: qa
-
-- name: pmm-ui-tests
-  branch: main
-  path: sources/pmm-ui-tests/src/github.com/percona/pmm-ui-tests
-  url: https://github.com/percona/pmm-ui-tests
-  component: qa
+  # COMMON
```
Comment (ademidoff) [inline ci-default.yml:2]: Sorry for the noice, it's the yaml linter who formatted the file.

File: ci.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+deps:
```
Comment (ademidoff) [inline ci.yml]: This will be reverted before merging, the rest - merged.

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-2404-7197473
client docker - perconalab/pmm-client-fb:PR-2404-7197473
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2404-7197473.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-2404-7197473&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2404-7197473.tar.gz

**Comment by JNKPercona** [comment]
UI tests have failed, Please check: UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/13264/

**Review by nikita-b — APPROVED** [review]

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-2404-23de259
client docker - perconalab/pmm-client-fb:PR-2404-23de259
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2404-23de259.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-2404-23de259&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2404-23de259.tar.gz

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/9575/

**Pull request merged**